controller/cmd/identity/main.go

```diff
@@ -42,6 +40,13 @@ func Main(args []string) {
 	addr := cmd.String("addr", ":8080", "address to serve on")
 	adminAddr := cmd.String("admin-addr", ":9990", "address of HTTP admin server")
 	kubeConfigPath := cmd.String("kubeconfig", "", "path to kube config")
+	controllerNS := cmd.String("controller-namespace", "", "namespace of the linkerd control plane")
```

Comment:
I think this is just specifies which namespace to emit kubernetes events to, right?  We could just call the flag `--namespace`.  or maybe even get the current namespace from the downward api.

Reply:
Hmm, I went with `--controller-namespace` as every other component also has it, and helps with consistency?